packages/core/admin/scripts/build.js

```diff
@@ -23,6 +23,7 @@ const buildAdmin = async () => {
    */
   const plugins = getPlugins([
     '@strapi/plugin-content-type-builder',
+    process.env.FEATURE_FLAG_CONTENT_RELEASES && '@strapi/content-releases',
```

Comment:
I thought the plugin could be uninstalled or disabled. Is that not the case? And if it is the case, how would this code behave?